src/utils/stream_decrypt.ts

```diff
@@ -0,0 +1,36 @@
+import { createDecipheriv } from 'crypto';
+import { Transform } from 'stream';
+import { CipherIV, FileKey } from '../types';
+
+const algo = 'aes-256-gcm'; // crypto library does not accept this in uppercase. So gotta keep using aes-256-gcm
+const authTagLength = 16;
+
+export class StreamDecrypt extends Transform {
```

Comment:
**For the reviewer**
See #148 for the latest change on this class

Reply:
This code was de-coupled from the CLI layer https://github.com/ardriveapp/ardrive-core-js/pull/83